### PR TITLE
Topic/dependencies

### PR DESCRIPTION
**Pull request by erikd-ambiata**
/cc @markhibberd @charleso @tmcgilchrist

### CODE REVIEW

File: boris.toml [hunk 1]
```diff
@@ -1,8 +1,10 @@
 [boris]
   version = 1
-[build.dist-7-8]
 [build.dist-7-10]
-[build.branches-7-8]
 [build.branches-7-10]
 [build.validate]
   command = [["validate-respect"]]
+[build.submodules]
```
Comment (charleso) [inline boris.toml]: FYI Not in the whitelist (either of them this time)

**Comment by charleso** [comment]
@erikd-ambiata Want to merge this in some form?

**Comment by erikd-ambiata** [comment]
I was hoping to have a boris validator before I had another run at this :/ .

**Comment by erikd-ambiata** [comment]
Went through this IRL with @charleso .

**Pull request merged**